### PR TITLE
AJout bagger cyrille hemidy

### PR DESCRIPTION
**Pull request by chemidy**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -16235,6 +16235,79 @@ var data = {
...
+          ],
+          "lang": [
+            "fr"
+          ]
+        }
+      ],
+      "location": "Nantes et environs",
+      "websites": [
+        {
+          "name": "Speaker Deck",
```
Comment (dadoonet) [inline baggers.js]: Please replace it with `Speakerdeck` instead as marked on the README page.

**Review by dadoonet — CHANGES_REQUESTED** [review]
Left a small comment

**Comment by chemidy** [comment]
Ok fixed, sorry for the misspelling

**Review by dadoonet — APPROVED** [review]

**Comment by dadoonet** [comment]
Welcome!

**Pull request merged**